src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -7,16 +7,25 @@
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    using Orleans.Providers;
+
+    public class OrleansJsonSerializer : IExternalSerializer
     {
+        public const string UseFullAssemblyNamesProperty = "UseFullAssemblyNames";
+        public const string IndentJsonProperty = "IndentJSON";
         private static JsonSerializerSettings defaultSettings;
         private Logger logger;
 
+        static OrleansJsonSerializer()
+        {
+            defaultSettings = GetDefaultSerializerSettings();
+        }
```

Comment:
I got confused, I guess it's OK... It's not configurable now, but that's orthogonal to this PR, merging